### PR TITLE
TB Treatment History Widget

### PR DESCRIPTION
**Pull request by semujju**
https://app.asana.com/0/1133167201254915/1199376249075581/f

### CODE REVIEW

File: omod/src/main/resources/apps/patient_dashboard_app.json [hunk 1]
```diff
@@ -247,5 +247,29 @@
       }
     }
   ]
+},{
```
Comment (sherrif10) [inline omod/src/main/resources/apps/patient_dashboard_app.json:250]: Thanks for working on this, On this line 250 i would do some identification

Comment (slubwama) [reply omod/src/main/resources/apps/patient_dashboard_app.json:250]: @sherrif10  kindly be more elaborate about your comment.

Comment (sherrif10) [reply omod/src/main/resources/apps/patient_dashboard_app.json:250]: Thanks @slubwama I meant to separate line 250 and add open curry brace no next line

Comment (semujju) [reply omod/src/main/resources/apps/patient_dashboard_app.json:250]: @sherrif10 this is noted, thanks.

Comment (semujju) [reply omod/src/main/resources/apps/patient_dashboard_app.json]: @sherrif10, this requested has been duly effected.

Comment (sherrif10) [reply omod/src/main/resources/apps/patient_dashboard_app.json]: @semujju  thanks  LGTM  though am seeing many deletions of config json files cc @slubwama can you confirm  thanks

**Pull request merged**